src/include/cephfs/libcephfs.h

```diff
@@ -251,8 +251,9 @@ int ceph_init(struct ceph_mount_info *cmount);
  *
  * An error will be returned if this libcephfs instance is already
  * mounted. This function is an alternative to setting the global
- * client_mds_namespace setting.  Using this function enables multiple
- * libcephfs instances in the same process to mount different filesystems.
+ * client_fs or client_mds_namespace(Deprecated) setting.  Using this
```

Comment:
let's drop the mention about `client_mds_namespace` too

Reply:
Sure.